htdocs/compta/facture/class/facture.class.php

```diff
@@ -5123,6 +5130,37 @@ public function sendEmailsRemindersOnInvoiceDueDate($nbdays = 0, $paymentmode =
 			return $error;
 		}
 	}
+
+	/**
+	 * See if current invoice date is posterior to the last invoice date among validated invoices of same type.
+	 * @return boolean
+	 */
+	public function willBeLastOfSameType()
+	{
+		// get date of last validated invoices of same type
+		$sql  = 'SELECT datef';
```

Comment:
For sql forging, please use ", the simple ' is reserved for string inside the sql (so string in sql format).
This will allow some of our security tool to works better against sql injection.

Reply:
Thanks @eldy , I corrected it. I think the PR can be merged.